default.env

```diff
@@ -1,6 +1,2 @@
 LOG_LEVEL=DEBUG
-USERID_HEADER_KEY=userid
-GROUPS_HEADER_KEY=usergroups
```

Comment:
why these env vars have been removed from the default?

Reply:
These variables were being used in the wrong way. At the moment we decided to use the de default values of these variables, otherwise we would have had to develop a configuration validator and a meta-schema to fit the custom values.